rpc/clients/typescript/test/ws_client_test.ts

```diff
@@ -173,7 +173,7 @@ blockchainTests.resets('WSClient', env => {
                 // timestamp is approximately equal (within 1 second) because the server
                 // will receive the request slightly after it is sent.
                 const now = new Date(Date.now()).getTime();
-                const perPage = 10;
+                const perPage = ordersLength / 2;
```

Comment:
This test wasn't actually making multiple paginated requests before so I modified it so that it does.